package.json

```diff
@@ -121,7 +121,10 @@
                 },
                 "args": [
                     "/debugAdapter",
-                    "/logLevel:Verbose"
+                    "/logLevel:Verbose",
+                    "/hostName:\"Visual Studio Code Host\"",
+                    "/hostProfileId:Microsoft.VSCode",
+                    "/hostVersion:0.5.0"
```

Comment:
It's the VS Code extension's version. There will be a different property in the runspace for getting the PSES version